usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -8,8 +8,8 @@
 from usaspending_api.references.models import PSC
 from usaspending_api.accounts.views.federal_accounts_v2 import filter_on
 from .filter_helpers import combine_date_range_queryset, total_obligation_queryset
-from usaspending_api.awards.models import FinancialAccountsByAwards
-from usaspending_api.awards.models_matviews import UniversalAwardView, UniversalTransactionView
+from usaspending_api.awards.models import FinancialAccountsByAwards, UniversalTransactionTableView
```

Comment:
I do not know why the materialized views were in their own model... perhaps just for organizational purposes.  By default, Django looks for models in a `models` file or package.  Since the materialized views are not managed, this hasn't really been a problem.